### ISSUE TITLE
Open entity graphs from entity explorer

### ISSUE THREAD

**Issue by trevore** [oculusinfo/TellFinder#31]
Analysts would like to be able to pull up the entity graph view of a potentially connected cluster found through the entity explorer in a new window without having to run another search based off of information present in the entity explorer.

PILOT-64

**Comment by eric-oculus** [comment]
Agreed. In the old graph, you could single click to select a cluster or double click to 'focus' on a different cluster. The focus operation should at least alter the URL so that the back button works. Even better would be to have a button which allowed opening in a new tab or the same tab.
